src/AccessibilityInsights/Scripts/UIAccess.cmd

```diff
@@ -0,0 +1,40 @@
+@echo off
+ if /i '%1' == 'Enable' goto Verify
+ if /i '%1' == 'Disable' goto Verify
+ 
+ :Usage
+ echo.
+ echo UIAccess.cmd
+ echo.
+ echo This script will set the UIAccess status of Accessibilty Insights for Windows.
+ echo UIAccess support is disabled by default. This script must be run from an
+ echo administrative command prompt.
+ echo.
+ echo Usage: UIAccess.cmd ^<action^>
+ echo.
+ echo Where ^<action^> is one of:
+ echo   ENABLE  (to enable UIAccess support)
+ echo   DISABLE (to disable UIAccess support)
+ echo.
+ goto :eof
+ 
+ :Verify
+ echo.
+ echo CAUTION! You should only change the UIAccess state if you understand the
+ echo          potential security implications!
```

Comment:
>potential security implications [](start = 15, length = 31)

This would be better pointing to a web page, but since we don't have one yet, there's no link